src/pages/index.html

```diff
@@ -1,29 +1,332 @@
+              class="testing-container--list--card d-flex justify-content-start align-items-start"
+            >
+              <div>
+                <span
+                  class="testing-container--list--card--icon d-flex justify-content-center align-items-center"
+                  style="background-color: {{ item.iconBackgroud }}; "
+                >
+                  {{ item.icon | safe }}
+                </span>
+              </div>
+
+              <div class="testing-container--list--card--details">
+                <h2 class="testing-container--list--card--details--title">
+                  {{ item.title }}
+                </h2>
+                <p class="testing-container--list--card--details--info">
+                  Many desktop publishing packages and web page editors now use
+                  for them.
+                </p>
+              </div>
+            </div>
+            {% endfor %}
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="about-section position-relative overflow-hidden">
+  <span class="quarter-circle position-absolute">
+    {{ vectors.quarterCircle() }}
+  </span>
+  <span class="right position-absolute"> {{ vectors.right() }} </span>
+
+  <div class="container">
+    <div class="about-container">
+      <div class="row gap-84 justify-content-lg-end justify-content-center">
+        <div class="col-lg-5 col-9">
+          <div
+            class="about-container--images-box d-flex justify-content-xl-end justify-content-center"
+          >
+            <div class="position-relative">
+              <img
+                src="../assets/images/main/image1.png"
+                class="about-container--images-box--main"
+              />
+              <div
+                class="about-container--images-box--video-box position-absolute d-flex justify-content-end"
+              >
+                <div class="position-relative">
+                  <img src="../assets/images/main/image2.png" />
+                  <span class="position-absolute">
+                    <svg
+                      xmlns="http://www.w3.org/2000/svg"
+                      width="61"
+                      height="62"
+                      viewBox="0 0 61 62"
+                      fill="none"
+                    >
+                      <g filter="url(#filter0_b_1_1295)">
+                        <circle
+                          cx="30.6916"
+                          cy="31.1803"
+                          r="30.2451"
+                          fill="#FEFEFF"
+                        />
+                      </g>
+                      <path
+                        d="M39.9082 30.5538C40.6279 30.9704 40.6268 32.0098 39.9062 32.4247L26.1102 40.37C25.3896 40.785 24.49 40.2643 24.4909 39.4328L24.5081 23.5124C24.509 22.6809 25.4098 22.1622 26.1294 22.5787L39.9082 30.5538Z"
+                        fill="#2B59FF"
+                      />
+                      <defs>
```

Comment:
If you won't add any effects to the svg you don't have to use it as code, just use it like any other image

Reply:
done.